Sources/Logger/Logger.swift

```diff
@@ -29,6 +29,16 @@ public struct Logger<Category: CategoryKey>: Loggerable {
 		return logger
 	}
 
+    /// - Parameters:
+    ///   - level: The log level that will be used for logged message.
+    ///   - access: Access level describing publiicity of the logged message.
```

Comment:
You have a mistake here.
Turn on Spelling and Grammar in the Xcode:
`Edit->Format->Spelling and Grammar->Check Spelling While Typing`